UCLALibrary/voyager-archive
# SYS-1000: Display invoices

Implements [SYS-1000](https://jira.library.ucla.edu/browse/SYS-1000).

This PR allows users to search by invoice number, and displays data from
* Invoice header
* Invoice adjustments (generally tax, freight/shipping, discounts)
* Invoice line items (what was purchased, via POs)

`InvoiceLineView` database view and model were tweaked to add title and status for display.  There's also new sample data sufficient for testing.

Testing: 
* Restart docker-compose system, if running
* Run: `sql_support/initialize_database.sh`
* Search for Invoice number: `UCLA-058A`
* Invoice header, 2 adjustments, and 3 invoice lines should display.

![invoice_UCLA-058A](https://user-images.githubusercontent.com/2213836/196573119-2dd630aa-b8dd-43da-bd6e-6c706aa537cc.png)
